django/db/models/sql/compiler.py

```diff
@@ -628,6 +602,36 @@ def as_sql(self, with_limits=True, with_col_aliases=False):
             # Finally do cleanup - get rid of the joins we created above.
             self.query.reset_refcounts(refcounts_before)
 
+    def get_for_update_part(self, with_limit_offset):
```

Comment:
`get_for_update_part()` -> `get_for_update_sql()`. We could move this hook in to a separate commit/PR.

Reply:
Changed.